core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1646,6 +1646,10 @@ public String getDisplayName() {
         return jdks;
     }
 
+    public void setJDKs(Collection<? extends JDK> jdks) {
```

Comment:
New public method without Javadoc.

Maybe make this `@Restricted(NoExternalUse.class)` and document that `JDK.setInstallations(…)` should be used instead?